src/scenes/AcceleratorRouter/ApproveDialog.tsx

```diff
@@ -0,0 +1,53 @@
+import { Typography, useTheme } from '@mui/material';
+import { Button, DialogBox } from '@terraware/web-components';
+
+import strings from 'src/strings';
+
+export type ApproveDialogProps = {
+  onClose: () => void;
+  onSubmit: () => void;
+};
+
+export default function ApproveDialog({ onClose, onSubmit }: ApproveDialogProps): JSX.Element {
```

Comment:
ApproveDeliverableDialog? And the filename.

Reply:
Good call 👍